phpBB/styles/prosilver/theme/common.css

```diff
@@ -1251,6 +1251,14 @@ ul.linklist:after,
 	margin-left: 58px;
 }
 
+.notification-alert,
+.compact .notification-alert {
+	border-radius: 50px;
+	font-size: 0.9em;
+	opacity: 0.8;
+	padding: 4px 7px;
```

Comment:
being nit picky but change 7 to 8, 4/8 is pretty standard across multiple frameworks for badges/tags/labels ;)